.circleci/config.yml

```
@@ -38,13 +38,13 @@ workflows:
   commit:
     jobs:
       - test
-  hourly:
+  threehourly:
     triggers:
       - schedule:
-          cron: 0 * * * *
+          cron: 30 0,3,6,9,12,15,18,21 * * *
           filters:
             branches:
               only:
-                - main
+                - stablise_flakey_test
```

Needs to be changed back to the `main` branch.

```suggestion
                - main
```